ui/css/module/_styleguide.scss

```diff
@@ -0,0 +1,26 @@
+.styleguide-container {
+  padding: 1rem;
+  max-width: 800px;
```

Comment:
Why do we need to limit the width here?

Reply:
@cmc333333 For the time being, the page is way too wide on a large screens. @rtwell is looking at a better way to style this, but it helps.